docs/src/components/Nav/Nav.js

```diff
@@ -47,7 +47,9 @@ const Nav = ({ mdxTree, posts }) => {
         const linkPath = `/${post.filePath.replace(/\.mdx?$/, '')}`;
 
         return {
-          label: formatLabel(post?.data?.title || post?.filePath),
+          label:
+            post?.data?.title ||
+            (post?.filePath && formatLabel(post?.filePath)),
```

Comment:
We were processing the human created title for the top pages which we can safely get directly from the page frontmatter